doc/source/ray-observability/ray-logging.rst

```diff
@@ -77,10 +111,17 @@ This produces the following output:
     (MyActor(index=2) pid=482120) hello there
     (MyActor(index=1) pid=482119) hello there
 
-How to set up loggers
+Customizing worker loggers
 ~~~~~~~~~~~~~~~~~~~~~
-When using ray, all of the tasks and actors are executed remotely in Ray's worker processes. 
-Since Python logger module creates a singleton logger per process, loggers should be configured on per task/actor basis. 
+When using ray, all of the tasks and actors are executed remotely in Ray's worker processes.
+
+If a user intends to log a message via a worker, a logger should be used. Typically the way this is done is by getting a logger using the filename where the log is generated; see `the Python logging documentation <https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial>`__ for more information about this.
```

Comment:
```suggestion
If you intend to log a message via a worker, use a logger. Get a logger using the filename of the generated log; see `the Python logging documentation <https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial>`__ for more information.
```

Reply:
Hmm, I've reworded slightly to avoid using 'log' too much here. Let me know if you'd prefer something different.

Reply:
Much better!